packages/time/src/types.d.ts

```diff
@@ -218,13 +218,17 @@ export type TimeMathType = {
   toRel: (rel: RelativeTime | number, brand?: TimerBrand) => RelativeTime;
   /**
    * An absolute time + a relative time gives a new absolute time.
+   *
+   * @template {Timestamp} T
    */
-  addAbsRel: (abs: Timestamp, rel: RelativeTime) => Timestamp;
+  addAbsRel: (abs: T, rel: RelativeTime) => T;
```

Comment:
Is type parameterization useful for these? Do you expect subtypes of `RelativeTime` and `Timestamp`? Why generalize the `rel*` parameters of `addRelRel` below but not the `rel` parameter of `addAbsRel` above?

Reply:
Aha! Thanks.